### DISCUSSION TITLE
Can I load an image based on dark/light mode?

### DISCUSSION THREAD

**Discussion by karpetrosyan** [squidfunk/mkdocs-material#7598] [Q&A]
Can I load an image (logo) based on dark/light mode. I have two logos one for light mode (white for dark mode) and (black for light mode) and I want to load it based on mode that is currently set, and change the logo when mode is changing

**Comment by alexvoss** [comment]
If it is an image in the content then the answer is you can, [see the documentation](https://squidfunk.github.io/mkdocs-material/reference/images/?h=dark+mo#light-and-dark-mode). If it is not then you'd need to be more specific.
